### ISSUE TITLE
Dependency inconsistency

### ISSUE THREAD

**Issue by opoudjis** [relaton/relaton-w3c#20]
````
Bundler could not find compatible versions for gem "rdf":
  In Gemfile:
    metanorma-nist was resolved to 2.0.7, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.6, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-w3c (~> 1.11.0) was resolved to 1.11.0, which depends on
              linkeddata (~> 3.1.0) was resolved to 3.1.6, which depends on
                rdf-normalize (~> 0.4) was resolved to 0.5.0, which depends on
                  rdf (~> 3.2)

    metanorma-nist was resolved to 2.0.7, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.6, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-w3c (~> 1.11.0) was resolved to 1.11.0, which depends on
              rdf (~> 3.1.0)
````

**Comment by ronaldtse** [comment]
@andrew2net it loos like Relaton-w3c is locked to rdf 3.1.0 but the linkeddata gem is locked to rdf 3.2. Looks best to relax the version of rdf.

**Comment by opoudjis** [comment]
Also:

````
Bundler could not find compatible versions for gem "sparql":
  In Gemfile:
    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-w3c (~> 1.11.0) was resolved to 1.11.0, which depends on
              sparql (~> 3.1.0)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-w3c (~> 1.11.0) was resolved to 1.11.0, which depends on
              linkeddata (~> 3.1.0) was resolved to 3.1.6, which depends on
                shex (~> 0.6, >= 0.6.4) was resolved to 0.7.1, which depends on
                  sparql (~> 3.2)
````

**Comment by opoudjis** [comment]
````
  In Gemfile:
    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-bipm (~> 1.11.0) was resolved to 1.11.0, which depends on
              serrano (~> 1.0) was resolved to 1.0.0, which depends on
                faraday_middleware (~> 1.0) was resolved to 1.2.0, which depends on
                  faraday (~> 1.0)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-calconnect (~> 1.11.0) was resolved to 1.11.0, which depends on
              faraday

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-ogc (~> 1.11.0) was resolved to 1.11.0, which depends on
              faraday (~> 1.1)
````

**Comment by opoudjis** [comment]
````
  In Gemfile:
    equivalent-xml (~> 0.6) was resolved to 0.6.0, which depends on
      nokogiri (>= 1.4.3)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        isodoc (~> 2.0.0) was resolved to 2.0.8.1, which depends on
          html2doc (~> 1.3.0) was resolved to 1.3.1, which depends on
            nokogiri (~> 1.12)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        isodoc (~> 2.0.0) was resolved to 2.0.8.1, which depends on
          nokogiri (~> 1.12.0)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-itu (~> 1.11.0) was resolved to 1.11.0, which depends on
              mechanize (~> 2.8.0) was resolved to 2.8.4, which depends on
                nokogiri (>= 1.11.2, ~> 1.11)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        metanorma-utils (~> 1.2.8) was resolved to 1.2.8, which depends on
          nokogiri (~> 1.11)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0) was resolved to 1.9.5, which depends on
          relaton (>= 1.9.8) was resolved to 1.11.0, which depends on
            relaton-ietf (~> 1.11.0) was resolved to 1.11.0, which depends on
              relaton-bib (~> 1.11.0) was resolved to 1.11.1, which depends on
                nokogiri (~> 1.12.5)

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        metanorma-utils (~> 1.2.8) was resolved to 1.2.8, which depends on
          sterile (~> 1.0.14) was resolved to 1.0.23, which depends on
            nokogiri (>= 1.11.7)
````

**Comment by opoudjis** [comment]
````
  In Gemfile:
    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        isodoc (~> 2.0.0) was resolved to 2.0.8.1, which depends on
          relaton-cli

    metanorma-nist was resolved to 2.0.8, which depends on
      metanorma-standoc (~> 2.0.0) was resolved to 2.0.7, which depends on
        relaton-cli (~> 1.9.0)
````

**Comment by andrew2net** [comment]
First two comments are fixed. Last three don't have collisions, they just need to update `bundle update`.
